fusor-ember-cli/app/controllers/assign-nodes.js

```diff
@@ -427,14 +429,9 @@ export default Ember.Controller.extend(DeploymentControllerMixin, NeedsDeploymen
   },
 
   disableAssignNodesNext: Ember.computed('unassignedRoles.[]', function() {
-    return (this.get('unassignedRoles.length') > 0);
-  }),
-
-  nextStepRouteNameAssignNodes: Ember.computed('isCloudForms', function() {
```

Comment:
This is no longer needed, since next step in OSP is the overcloud route. It's not dynamic.